test-app/build-tools/android-metadata-generator/src/src/com/telerik/metadata/Builder.java

```diff
@@ -238,13 +246,13 @@ private static void setMethodsInfo(TreeNode root, TreeNode node, NativeClassDesc
                         if (existingNodeMethods.containsKey(sig)) {
                             continue;
                         }
-                        node.instanceMethods.add(mi);
+                        node.addInstanceMethod(mi);
                     }
                 }
 
-                if (mi.isResolved) {
-                    node.resolvedMethods.add(mi);
-                }
+                /* if (mi.isResolved) {
+                    node.addResolvedMethod(mi);
+                } */
```

Comment:
What were "resolved" methods? What's the impact of removing them?

Reply:
Oh, yeah, extra UNUSED code that was added by the original team -- I commented it out initially to verify it wasn't used by anything.  I should strip it out...